examples/lock-app/efr32/src/AppTask.cpp

```diff
@@ -53,23 +61,27 @@
 #include "wfx_host_events.h"
 #include <app/clusters/network-commissioning/network-commissioning.h>
 #include <platform/EFR32/NetworkCommissioningWiFiDriver.h>
-#endif
+#endif /* SL_WIFI */
 
 #define FACTORY_RESET_TRIGGER_TIMEOUT 3000
 #define FACTORY_RESET_CANCEL_WINDOW_TIMEOUT 3000
 #define APP_TASK_STACK_SIZE (4096)
 #define APP_TASK_PRIORITY 2
 #define APP_EVENT_QUEUE_SIZE 10
+#define EXAMPLE_VENDOR_ID 0xcafe
```

Comment:
this can be removed. It is not used and also invalid vendor id

Reply:
removed.